CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -29,6 +30,12 @@
     public DistanceSensor leftDistanceSensor;
     public DistanceSensor rightDistanceSensor;
     
+    public XEncoder leftDriveEncoder;
```

Comment:
Tiona is still doing the right thing - what I have is just to quickly verify their behavior. It means there may be a messy merge, but I wanted to make sure we immediately were able to verify encoder behavior.